client/src/components/login/login.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {NewUser} from '../newUser/newUser.jsx'
+import './login.sass';
+
+export class Login extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			username: '',
+			password: '',
+			showing: "login"
+		}
+		this.handleUsernameChange = this.handleUsernameChange.bind(this);
+		this.handlePasswordChange = this.handlePasswordChange.bind(this);
+		this.returnToLogin = this.returnToLogin.bind(this);		
+		this.handleSubmit = this.handleSubmit.bind(this);
+		this.handleClick = this.handleClick.bind(this);
+		this.updateAppUserState = this.updateAppUserState.bind(this);
+		this.goHome = this.goHome.bind(this);
+	};
+
+	goHome() {
+		this.props.return();
+	};
+
+	handleUsernameChange(event) {
+		this.setState({username: event.target.value});
+	};
+
+	handlePasswordChange(event) {
+		this.setState({password: event.target.value});
+	};
+
+	updateAppUserState() {;
+		console.log("Telling App to update State")
+		this.props.onLogin();
+	};
+
+	handleSubmit(event) {
+		event.preventDefault();
+
+		const user = {
+			username: this.state.username,
+			password: this.state.password
+		};
+
+		const userInfo = {
+			method: 'post',
+			// credentials: 'include',
+			headers: {
+				'Content-Type': 'application/json'
+			},
+			body: JSON.stringify(user)
+		};
+
+		fetch('auth/v1/login', userInfo)
+		.then(() => {
+			console.log("Logged in!");
+			localStorage.setItem('user', this.state.username);
+			this.setState({
+				username: '',
+				password: ''
+			});
+			this.updateAppUserState();
+			this.goHome();
+		})
+		.catch(err => {
+			console.log("Error logging user in");
+			console.log(err);
+		});
+
+	}
+
+	handleClick() {
+		// Open up newUser page
+		console.log("Show New User")
+		this.setState({
+			showing: "newUser"
+		})
+	}
+
+	returnToLogin() {
```

Comment:
This function is to pass to the New User component as a prop. From there you are returning to the login page.

As part of the login component, it could be renamed to "showLogin"

Reply:
Honestly, I kind of liked returnToLogin more than showLogin